src/promptflow/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## v1.17.1 (2025.1.13)
+
+### Bugs Fixed
+- Marshmallow 3.24 was recently released, removing the `_T` import, which caused a breaking change in Promptflow. We've eliminated the dependency on `_T` to resolve this issue.
+
+### Others
+- Promptflow Tracing feature is now disabled by default, with `PF_DISABLE_TRACING` set to true by default.
```

Comment:
Do we need to add this for `promptflow-devkit` or `promptflow-core` as well?

Reply:
Updated in the sub packages accordingly.